internal/clients/nightfall/nightfall.go

```diff
@@ -30,6 +32,13 @@ const (
 
 	defaultTimeout           = time.Minute * 20
 	MaxConcurrentRoutinesCap = 50
+	maxRetries               = 5
```

Comment:
nit - do you mind adding comments for each of these vars? We should have caught this in previous PR's for defaultTimeout and MaxConcurrentRoutinesCap. Since these are configuration related and pretty important to the functioning of htis client, it's always good to be extra clear about what's going on.